packages/compass-generative-ai/src/components/ai-signin-modal.tsx

```diff
@@ -102,7 +102,7 @@ const AISignInModal: React.FunctionComponent<SignInModalProps> = ({
 export default connect(
   (state: AtlasSignInState) => {
     return {
-      isSignInModalVisible: state.isModalOpen,
+      isSignInModalVisible: state.signInReducer.isModalOpen,
       isSignInInProgress: state.state === 'in-progress',
```

Comment:
Should this also be updated now that we have multiple reducers?
```suggestion
      isSignInInProgress: state.signInReducer.state === 'in-progress',
```